src/actions/index.js

```diff
@@ -1 +1,22 @@
+import axios from 'axios';
+import endPoints from '../containers/urls';
 
+export const storeCategories = () => {
+  return dispatch => {
+    axios.get(endPoints.getCategories)
+    .then(function (response) {
+      const categories  = response.data.categorys.results;
+      dispatch(setCategories(categories));
+    })
+    .catch(function (error) {
+      console.log(error);
```

Comment:
@Stanley-Okwii I don't think it is good practice to console.log error messages

Reply:
@walimike , Done